docs/source/user_guide/automl.ipynb

```diff
@@ -45,11 +45,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
```

Comment:
Can you `Restart Kernel and Clear Output` before merge?

Reply:
The problem lies in me forgetting to hit save one last time before adding and committing :P. I'm still not used to ipynb in the codebase.